### ISSUE TITLE
ATM namelists and data files changes required for using ulm_201505 patch forward

### ISSUE THREAD

**Issue by nikizadehgfdl** [NOAA-GFDL/MOM6-examples#37]
In ulm patch (ulm_201505) the ATM radiation input files were renamed in the ATM code (hard-coded).
In order to use this patch and forward  please copy the following files to somewhere accessible to mom6 examples so that they  can
be used when we want to switch to use  ulm_201505 forward. 

/lustre/f1/unswept/Niki.Zadeh/archive/input/ulm/ulm_201505_BW_new_sea_esf_aug2013/cns*

These are the same old cns files, only renamed. They don't cause answer changes (if we pick up
all the required namelists mods). I just checked this for MOM6_CM3G 

Also, the following namlists changes are required:

```
     <namelist name="cloud_spec_nml">
       cloud_type_form = 'strat'
     </namelist>
     <namelist name="random_number_streams_nml">
	do_legacy_seed_generation = .true. ! siena_201303
        force_use_of_temp_for_seed=.false.
     </namelist>
        <namelist name="radiation_driver_nml">
       rad_time_step= 10800,
       rad_package = 'sea_esf',
       do_clear_sky_pass=.true.,
       renormalize_sw_fluxes=.true.,
       zenith_spec = 'diurnally_varying'
       use_co2_tracer_field = .true.
       using_restart_file = .false.
     </namelist>
     <namelist name="radiation_driver_diag_nml">
      all_step_diagnostics = .true.,
     </namelist>
         <namelist name="radiative_gases_nml">
       verbose = 3
       gas_printout_freq = 240

       time_varying_co2 = .false.,! keep false for historical co2 to avoid fatal error message: rad co2 will be overidden
       co2_variation_type = 'linear',
       co2_dataset_entry = 1990, 1, 1, 0, 0, 0,
       co2_specification_type = 'time_series',
       co2_floor = 100.0E-06,
       co2_ceiling = 1600.0E-06,
       co2_data_source = 'input'

       time_varying_ch4 = .false.,
       ch4_variation_type = 'linear'
       ch4_dataset_entry = 1990, 1, 1, 0, 0, 0,
       ch4_specification_type = 'time_series'
       ch4_data_source = 'input'

       time_varying_n2o = .false.,
       n2o_variation_type = 'linear'
       n2o_dataset_entry = 1990, 1, 1, 0, 0, 0,
       n2o_specification_type = 'time_series'
       n2o_data_source = 'input'

       time_varying_f11 = .false.,
       f11_variation_type = 'linear'
       f11_dataset_entry = 1990, 1, 1, 0, 0, 0,
       f11_specification_type = 'time_series'
       f11_data_source = 'input'

       time_varying_f12 = .false.,
       f12_variation_type = 'linear'
       f12_dataset_entry = 1990, 1, 1, 0, 0, 0,
       f12_specification_type = 'time_series'
       f12_data_source = 'input'

       time_varying_f113 = .false.,
       f113_variation_type = 'linear'
       f113_dataset_entry = 1990, 1, 1, 0, 0, 0,
       f113_specification_type = 'time_series'
       f113_data_source = 'input'

       time_varying_f22 = .false.,
       f22_variation_type = 'linear'
       f22_dataset_entry = 1990, 1, 1, 0, 0, 0,
       f22_specification_type = 'time_series'
       f22_data_source = 'input'

       calc_co2_tfs_on_first_step = .false.,
       calc_co2_tfs_monthly = .true.,
       co2_tf_time_displacement = 0.0,

       calc_ch4_tfs_on_first_step = .true.,
       calc_ch4_tfs_monthly = .false.,
       ch4_tf_time_displacement = 0.0,

       calc_n2o_tfs_on_first_step = .true.,
       calc_n2o_tfs_monthly = .false.,
       n2o_tf_time_displacement = 0.0,

     </namelist>
     <namelist name="shortwave_driver_nml">
       do_cmip_diagnostics = .true.,
       swform = 'esfsw99'
       time_varying_solar_constant = .false.,
       solar_dataset_entry = 1990,1,1,0,0,0,
      </namelist>

     <namelist name="sealw99_nml">
       do_thick = .false.,
       do_nlte = .false.,
       do_lwcldemiss = .true.,
       continuum_form = 'ckd2.1',
       linecatalog_form = 'hitran_2000',
       verbose = 5
     </namelist>
```

**Comment by adcroft** [comment]
When I try using ulm_201515 (and after patching all the broken bits) I get to this FATAL message in CM2G63L:
> FATAL from PE    11: mg_drag_init: source_of_sgsmtn="computed", but topography data file does not exist

Which suggests either a new input file is needed or the namelist parameters for mg_drag have also changed?

**Comment by adcroft** [comment]
My bad - @jkrasting found my mistake. When bringing in the new files I clobbered the navy topography.

**Comment by adcroft** [comment]
Should have been closed by 1f36497c96e296dac9293bb995484306e4148f1d
